app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/exemptionandlosses/in_year_losses.scala.html

```diff
@@ -131,7 +132,8 @@
             )),
             customErrorKey = Some(s"$key$userKey")
         )
-        <p>@Html(messages(s"$key$userKey.link", appConfig.cgtLossesUrl))</p>
+
+        <p>@Html(messages(s"$key$userKey.link", appConfig.personalRepresentativeUrl))</p>
```

Comment:
should the original link location change if it's not period of admin?

Reply:
there was no such logic to differentiate based on `userType` before so I assumed it would still be the case for this piece of work - not sure to be honest as the I didn't get any indication from the JIRA ticket to give any steer on this

Reply:
I think the intention was that the previous content that was there stays as it is and the ticket just describes changes when it's period of admin

Reply:
refined the logic to only apply this for period of admin